scripts/extract_vista_cliente_expressions.py

```
@@ -0,0 +1,227 @@
+        for case in tp.get("cases") or []:
+            if isinstance(case, dict):
+                _walk_activities(case.get("activities") or [], pipeline_name, out, depth, path)
+
+
+def _extract_all(corpus_dir: Path) -> list[dict]:
+    records: list[dict] = []
+    for fp in sorted(corpus_dir.glob("*.json")):
+        try:
+            d = json.loads(fp.read_text())
+        except Exception:
+            continue
```

Ruff (0.15.10)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)